TYPO3.Flow/Classes/TYPO3/Flow/I18n/Service.php

```diff
@@ -279,6 +281,9 @@ protected function generateAvailableLocalesCollectionByScanningFilesystem()
             $directories = array(Files::getNormalizedPath($packageResourcesPath));
             while ($directories !== array()) {
                 $currentDirectory = array_pop($directories);
+                if ($blacklistPattern !== '' && preg_match('#' . str_replace('#', '\#', $blacklistPattern) . '#', $currentDirectory) > 0) {
```

Comment:
They had no boolean return values leftover on the day they implemented the 'preg_match' method in PHP. *facepalm*